conda/environments/cudf_dev_cuda11.2.yml

```
@@ -66,3 +66,4 @@ dependencies:
       - git+https://github.com/dask/distributed.git@main
       - git+https://github.com/python-streamz/streamz.git@master
       - pyorc
+  - ptxcompiler
```

@jakirkham do we need this here too? Similar to your suggestion in https://github.com/rapidsai/integration/pull/389?

```suggestion
  - ptxcompiler  # [linux64]
```

---

won't this break WSL compat ?

---

Don't think there are selectors in environment files. Not yet at least ( https://github.com/conda/conda/issues/8089 )

Idk how does `ptxcompiler` work for WSL?

---

Maybe let’s forgo that discussion for now. Apologies for the distraction

---

Should we merge in these changes?

---

No worries

I think Graham raised a concern above, which I don't think was addressed. Made some suggestions below ( https://github.com/rapidsai/cudf/pull/9687#pullrequestreview-809315191 ), which I think should address them, but Graham can hopefully let us